go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/hertz-contrib/csrf
+
+go 1.18
+
+require (
+	github.com/cloudwego/hertz v0.3.2
```

Comment:
update hertz-version

Reply:
It was aligned with other extension libraries, so it was removed.

Reply:
> update hertz-version

done